mix.exs

```diff
@@ -76,4 +77,12 @@ defmodule ExCoveralls.Mixfile do
       }
     ]
   end
+
+ defp xref() do
```

Comment:
FWIW, I think a better choice would be to change the `optional: true` for `castore` to something like `function_exported?(:public_key, :cacerts_load, 0)`.

Reply:
True, this could be another way to fix this issue, but I preferred a solution that didn't require modifying the code.
Also, this is when I saw [this PR in ex_aws](https://github.com/ex-aws/ex_aws/pull/1128) that I figured this would be a good way.